frontend/src/Components/DescriptionBox.js

```diff
@@ -1,21 +1,28 @@
-import { Container, Card } from "react-bootstrap";
+import { Container, Card, Stack } from "react-bootstrap";
 
-export default function DescriptionBox() {
+export default function DescriptionBox({
+  children,
+  title,
+  imgSrc,
+  height,
+  width,
+  fluid
+}) {
   return (
     <>
-      <Container fluid>
-        <Card bg="info">
-          <Card.Body>
-            <Card.Title>What is CS Catalog?</Card.Title>
-            <Card.Text>
-              CS Catalog is a helpful site where we are seeking to faciliate
-              understanding and exposure to Machine Learning (ML), which has
-              been an incredibly popular discripline these days. We are aiming
-              at assisting those students who are uncertain where to start in
-              ML. Our target users will be Computer Science students who are
-              passionate about ML or are planning to get into this area.
-            </Card.Text>
-          </Card.Body>
+      <Container fluid={fluid}>
+        <Card bg="info" className="rounded-5">
+          <Stack direction="horizontal">
+            <Card.Img
+              src={imgSrc}
+              style={{ width: width + "em", height: height + "em" }}
+              className="p-1 rounded-5"
+            />
+            <Card.Body>
+              <Card.Title>{title}</Card.Title>
+              <Card.Text>{children}</Card.Text>
+            </Card.Body>
+          </Stack>
```

Comment:
This looks like you're including implementations for a separate task in this PR. Please separate them into other branches and set up PRs for those tasks accordingly. Also, I am pretty sure this task is being taken up by someone else so I would check in with them if you did their work for them already.